ui-v2/tests/acceptance/components/acl-filter.feature

```diff
@@ -25,12 +25,12 @@ Feature: dc / components /acl filter: Acl Filter
 
     When I click all on the filter
     Then I see allIsSelected on the filter
-    Then I type with yaml
+    Then I fill in with yaml
```

Comment:
Was `fill in` already in the steps (assuming yes)? I didn't see it added in this PR.

Reply:
Yeah, its a funny one (well one of those funny ones that's not that funny 😄 ), I made a 'type with' step cos i couldn't get '`fillIn` to work. But of course `fillIn` wasn't working as I was using `onkeyup` instead of `oninput` on `input`-likes, ouch! 😄

Reply:
Here's 'fill in' bits if you want a peek:

https://github.com/hashicorp/consul/blob/7f35ca33f4a2753f6d6e7a7b635e1594f977c38d/ui-v2/tests/steps.js#L108-L112

Reply:
Ah yeah, with onkeyup you have to trigger the event which is gnarlier - now I'm wondering if the console input in vault has this same issue... 🤔